src/main/resources/public/js/app.js

```diff
@@ -9,11 +9,20 @@ $( document ).ready(function() {
    });
 });
 
-
 var state = "fire";
 var lastClickAcross;
 var lastClickDown;
 
+function startGame(mode) {
+	if (mode == "hard") {
+		
+	} else if (mode == "easy") {
+		
+	}
+	
```

Comment:
So just to clarify here - the JS code is being implemented once the back end changes have been made? None of it exists yet? Makes sense - just wanted to be certain.